<launch>

    <node pkg="create_fundamentals" type="driver.py" name="driver" output="screen"/>

    <include file="$(find create_fundamentals)/laser.launch" />

    <node pkg="green_fundamentals" type="publish_map.py" name="publish_map" output="screen" />

    <node pkg="green_fundamentals" type="map_generator" name="map_generator" output="screen" />

    <node pkg="green_fundamentals" type="obstacle_detector" name="obstacle_detector" output="screen" />

    <node pkg="green_fundamentals" type="costume_server.py" name="costume" output="screen" />

</launch>
